src/caret_analyze/plot/plot_facade.py

```diff
@@ -188,7 +188,7 @@ def create_response_time_timeseries_plot(
             This also accepts multiple inputs by unpacking.
         case: str, optional
             Response time calculation method, best by default.
-            supported case: [best/worst/all].
+            supported case: [best/worst/worst_in_input/all].
```

Comment:
```suggestion
            supported case: [best/worst/worst-in-input/all].
```

Reply:
Fixed in https://github.com/tier4/CARET_analyze/pull/336/commits/318c038007dfb0f14bd6648fe8dedc6824680d2a.

Reply:
Not fixed.